apps/web/server/routers/viewer.tsx

```diff
@@ -574,8 +574,8 @@ const loggedInViewerRouter = createProtectedRouter()
       // `flatMap()` these work like `.filter()` but infers the types correctly
       const conferencing = apps.flatMap((item) => (item.variant === "conferencing" ? [item] : []));
       const payment = apps.flatMap((item) => (item.variant === "payment" ? [item] : []));
+      const other = apps.flatMap((item) => (item.variant.startsWith("other") ? [item] : []));
       const calendar = apps.flatMap((item) => (item.variant === "calendar" ? [item] : []));
-      const other = apps.flatMap((item) => (item.variant === "other" ? [item] : []));
```

Comment:
It was already there in the `main` branch so I replaced it with the `startsWith` implementation instead for the reason I explained.